extensions/functions_aggregate_decimal.yaml

```diff
@@ -0,0 +1,41 @@
+%YAML 1.2
+---
+aggregate_functions:
+  - name: "count"
```

Comment:
overloading functions based only on return type causes problems, see e.g. https://github.com/substrait-io/substrait/pull/606#issuecomment-1964054841

Reply:
I have them in a different yaml file. can this still cause problems?

Reply:
different files should be ok.

Reply:
I believe it'll still break substrait-java, see https://github.com/substrait-io/substrait-java/issues/275

Reply:
I think we should fix that in substrait-java then as opposed to holding back the specification. Extensions being independent is key to extensions being extensions :)

Reply:
It's possible for us to work around duplicates in substrait-java by excluding them from the default extensions for now. See https://github.com/substrait-io/substrait-java/issues/275#issuecomment-2257176686